integration/container/mounts_linux_test.go

```diff
@@ -426,6 +426,69 @@ func TestContainerCopyLeaksMounts(t *testing.T) {
 	assert.Equal(t, mountsBefore, mountsAfter)
 }
 
+func TestContainerBindMountReadOnlyDefault(t *testing.T) {
```

Comment:
I think we need a to check if rro is supported on the kernel for the test to function properly.

Alternatively, we could just check the registered mountpoints in the API to see if the option is set as expected.

Reply:
Also sorry, one more check, test should be skipped if api version is < 1.44.

Reply:
Why? It also tests that the behavior doesn't change when <1.44 API is requested.

Reply:
So yes technically the 1.43 test should pass on an older daemon.
But the 1.44 test would fail since it is not a supported version.

Reply:
1.44 test will be skipped: https://github.com/moby/moby/pull/47391/files#diff-0ac47068153ec59b77b2044bd459fb6395f028e6e112dc03645f7ba72f324bf2R456

But yeah the `version: ""` would fail, updated it to be also skipped.

Also, added the test for `api.MinSupportedAPIVersion` client version.